content/en/tracing/trace_search_and_analytics/_index.md

```diff
@@ -129,7 +131,7 @@ Retained Analytics is available from the same page as Live Analytics.  To switch
 
 {{< img src="tracing/live_search/HistoricalAnalytics2.gif" alt="Historical Analytics" >}}
 
-All spans indexed by retention filters or legacy App Analytics filters are available to be searched when using trace analytics. These spans are kept by Datadog for 15 days after being indexed by a retention filter.
+All spans indexed by _custom_ retention filters (not the intelligent retention filter) or legacy App Analytics filters are available to be searched when using trace analytics. These spans are kept by Datadog for 15 days after being indexed by a retention filter.
```

Comment:
Not sure we should mention "legacy App Analytics filters" as this is actually a custom retention filter (the only difference is that it is created automatically when users start sending spans marked as analyzed)